dartagnan/src/main/java/com/dat3m/dartagnan/verification/solving/RefinementSolver.java

```diff
@@ -86,6 +89,10 @@ public class RefinementSolver extends ModelChecker {
     private static final String FRE = "fre";
     private static final String POLOC = "po-loc";
 
+    private final ExecutionModelManager nextModelManager = new ExecutionModelManager();
```

Comment:
Why does this need to be an instance member? I see only one method that uses it and that one could just generate the manager locally.